README.md

```diff
@@ -7,3 +7,9 @@
 [![codecov.io](https://codecov.io/github/DotNetAnalyzers/AsyncUsageAnalyzers/coverage.svg?branch=master)](https://codecov.io/github/DotNetAnalyzers/AsyncUsageAnalyzers?branch=master)
 
 This repository contains analyzers for best practices related to asynchronous programming. Where possible, code fixes are also provided to simplify the process of correcting violations.
+
+[Available on NuGet](https://www.nuget.org/packages/AsyncUsageAnalyzers/1.0.0-alpha003)
```

Comment:
:bulb: Can you change this to not specify the version? That will allow us to reference the package without constantly updating the file. 👍